templates/4b_normalize_image_data.ipynb

```diff
@@ -0,0 +1,142 @@
+{
```

Comment:
<p>Just to be explicit, I think having a <code>norm_json_path</code> variable passed here would re-enforce the sweep normalization step in the 1a notebook; it isn't immediately clear where the normalization parameters are coming from otherwise.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/toffy/pull/125/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
Added a description at the top of the notebook instead